py/desitarget/mock/data/select-mock-targets.yaml

```diff
@@ -28,15 +28,18 @@ targets:
     }
     LYA: {
         target_type: QSO,
-        mockfile: '{DESI_ROOT}/mocks/lya_forest/london/v4.2.0/master.fits',
+        mockfile: '{DESI_ROOT}/mocks/lya_forest/london/v9.0/v9.0.0/master.fits',
         format: CoLoRe,
         nside_lya: 16,
         zmin_lya: 1.8,
         density: 50,
         use_simqso: True,
-        balprob: 0.0,
-        add_dla: False,
-    }
+        sqmodel: 'default',  #Other options are 'lya_simqso_model_develop' and 'lya_simqso_model' (same as quickquasars)
+        balprob: 0.16,
+        add_dla: True,
+        add_metals: 'all',
+        add_lyb: True
+        }
     MWS_MAIN: {
```

Comment:
We use this "default" parameter file for the `minitest` integration tests [here](https://github.com/desihub/desitest/blob/master/mini/select-mock-targets.yaml) (which will need to be updated), so in the interest of compute time I've typically left off these additional features (BALs, DLAs, etc.).  Of course someone doing a simulation is welcome to use them. 

I could go either way, however.

Reply:
Leaving them optional is ok, I'll change it...